forge-gui/res/cardsfolder/upcoming/gaeas_gift.txt

```diff
@@ -0,0 +1,7 @@
+Name:Gaea's Gift
+ManaCost:1 G
+Types:Instant
+A:SP$ PutCounter | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature | CounterType$ P1P1 | CounterNum$ 1 | SubAbility$ DBPump | SpellDescription$ Put a +1/+1 counter on target creature you control. It gains reach, trample, hexproof, and indestructible until end of turn. (It can't be the target of spells or abilities your opponents control. Damage and effects that say "destroy" don't destroy it.)
+SVar:DBPump:DB$ Pump | Defined$ Targeted | KW$ Hexproof & Indestructible & Trample & Reach
+DeckHas:Ability$Counters & Keyword$Reach|Trample|Hexproof
```

Comment:
might as well add Indestructible here?

not sure these keyword clues are super helpful for the AI? @Agetian any insight?